lib/QueryRunner.js

```diff
@@ -10,7 +10,8 @@ const defaultOptions = {
   portCache: true,
   noBreadthOrder: false,
   ipFamily: 0,
-  requestPlayers: true
+  requestPlayers: true,
+  rejectUnauthorized: true,
```

Comment:
We can remove this as it isn't a general option, the default value will be set inside the protocol.

Reply:
Removed defaulting in QueryRunner.js and added default parameter in protocol file.